pages/introduction.vue

```diff
@@ -54,9 +51,65 @@
         transition.value = "slide-next";
         step.value++;
     };
+
+    setTimeout(function(){if(step.value == 0){nextStep();}},8000);
+
 </script>
 
 <style>
+
+.move-in1{
+    animation: movein 2s linear;
+}
+
+.move-in2{
+    animation: moveindelayed 4s linear;
+}
+
+.fade{
+    animation: fadein 5s linear;
+}
+
+@keyframes movein{
+    0%{
+        opacity: 0;
+    }
+    70%{
```

Comment:
die Zwischenschritte brauchst du in diesem fall nicht oder? Eigentlich sollte es ja reichen, die translation bei 0% zu definieren

Reply:
So wie ich es gemacht habe, werden sie benötigt. Die Zwischenschritte sind hier für das "Timing" verantwortlich. Als ich mit animation-delay und einer "Start"-opacity von 0 gerabeitet habe, hatte ich das Problem, dass die opacity nach der animation wieder auf 0 gesetzt wurde (was ich nicht erklären kann, aber so wie es jetzt ist, funktioniert es)